src/libopenrave/kinbodygrab.cpp

```diff
@@ -350,6 +350,9 @@ bool KinBody::Grab(KinBodyPtr pGrabbedBody, LinkPtr pGrabbingLink, const std::se
     OPENRAVE_ASSERT_FORMAT(pGrabbingLink->GetParent().get() == this, "env=%s, pGrabbingLink name='%s' for grabbing '%s' is not part of body '%s'", GetEnv()->GetNameId()%pGrabbingLink->GetName()%pGrabbedBody->GetName()%GetName(), ORE_InvalidArguments);
     OPENRAVE_ASSERT_FORMAT(pGrabbedBody.get() != this, "env=%s, body '%s' cannot grab itself", GetEnv()->GetNameId()%pGrabbedBody->GetName(), ORE_InvalidArguments);
 
+    // Flag that the body we're grabbing has been grabbed and may be tracked by more collision checkers than just the env checker
+    pGrabbedBody->_wasEverGrabbed = true;
```

Comment:
@snozawa @kanbouchou can you take a look at this usage of  `_wasEverGrabbed`? Thanks